Extension/package.json

```diff
@@ -2832,7 +2832,7 @@
       "binaries": [
         "./LLVM/bin/clang-format.darwin"
       ],
-      "integrity": "62A2ECB43511129F920D87BAE9230A803B54712038BCCB485F8F10B84B1A2000"
+      "integrity": "AEB24D394118EAD87087DCC651F2EE329FC335ECE88ED6C8C2A9E93ED60DFECD"
```

Comment:
You can't update the hashes without also creating a new fwlink or else the download will fail (assuming you update an existing fwlink)

Reply:
There are no existing customers -- no online vsix shipped.